source/framework/tools/src/TRestTools.cxx

```diff
@@ -130,7 +130,7 @@ void TRestTools::LoadRESTLibrary(bool silent) {
             if (excluded) {
                 continue;
             }
-            fileList.emplace_back(it.path());
+            fileList.emplace_back(it.path().string());
```

Comment:
It is not necessary to add `.string()` since it will automatically convert it, but this is a matter of opinion on which way it looks more clean.

Reply:
If not add `.string()` there will be problem compiling under windows.

Reply:
> If not add `.string()` there will be problem compiling under windows.

Oh, I didn't know, good to know.